application/src/main/java/md/maib/retail/application/register_newcampaign/RegisterCampaignValidate.java

```diff
@@ -0,0 +1,63 @@
+package md.maib.retail.application.register_newcampaign;
+
+import am.ik.yavi.builder.ValidatorBuilder;
+import am.ik.yavi.core.ConstraintViolations;
+import am.ik.yavi.core.Validator;
+import io.vavr.control.Either;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+import md.maib.retail.model.campaign.CampaignMetaInfo;
+import md.maib.retail.model.campaign.CampaignState;
+import md.maib.retail.model.campaign.LoyaltyEventType;
+import md.maib.retail.model.conditions.Rule;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static io.vavr.control.Either.left;
+import static io.vavr.control.Either.right;
+
+@Getter
+@RequiredArgsConstructor  // Removed AccessLevel.PRIVATE
+@ToString
+public final class RegisterCampaignValidate {
```

Comment:
Rename to RegisterCampaign.
This is command for RegisterCampaignUseCase
creation it this validation is a implementation detail